src/components/crownConfig.vue

```diff
@@ -109,12 +111,19 @@ export default {
   computed: {
     style() {
       const { x, y, width: shapeWidth } = this.shape.findView(this.paper).getBBox();
-      const width = (this.node.diagram.bounds.width * this.paper.scale().sx);
-      const widthOffsetFromLabel = shapeWidth - width > 4 ? (shapeWidth / 2) - 15 : 0;
+      let width = (this.shape.getBBox().width * this.paper.scale().sx) - 20;
+      const widthOffsetFromLabel = shapeWidth - width > 3 ? (shapeWidth / 2) - 15 : 0;
```

Comment:
The numbers in the calculation are pretty 'magical'. Could this be cleared up by extracting them to constants and giving them names that represent what they're for?

Reply:
@JCook21 I removed the need for these constants in my PR, here https://github.com/ProcessMaker/modeler/pull/829/files#diff-c1b56e458c9fac3f4960d9c517f46cdbR195.

I think it's best to wait for that to be merged into `develop`, then this branch can be brought up to speed and use that logic here as well.

Reply:
@chipit24 good plan, thanks.